e2e/src/common.rs

```diff
@@ -95,6 +95,8 @@ pub fn wait_for(web3: &Web3<Http>, seconds: u32) {
         .execute("evm_mine", vec![])
         .wait()
         .expect("Cannot mine to increase time");
+    // TODO vk:
+    std::thread::sleep(std::time::Duration::from_secs(seconds as u64 + 10));
```

Comment:
Does anyone have a better idea for this? I don't like making the Ganache test run for 10 minutes but I don't see a way around it unless we add extra code to the scheduler just so the ganache test works.

Reply:
Why is this needed? Because we sleep for the entire time of the batch in one go? Maybe it's fine to break the scheduler into multiple smaller sleeps?

If we decide to remove the code that is checking for the most up to date batch Id and only rely on system time, we might want to implement our own SystemTime abstraction that on ganache we can replace with an object that uses the blockchain time.

Reply:
I am still looking into this part. The problem is what you mentioned and that ganache does not advance time at all unless `evm_increaseTime` is called. So if you have the `docker-compose stablex` running for 15 minutes the expected current batch id in the scheduler is `x + 3` while in Ganache it is still `x`. 
I'll see if I can `evm_increaseTime` the proper amount at the start of the test.

Reply:
As far as I know, ganache increases time one each evm_mine so it should advance whenever a transaction is sent (even if it was manipulated to be in the future by using evm_increaseTime). In case there is no transaction I guess there is also no need to run the driver. 
There is still the problem of scheduling a long sleep based on an old time, but we could break the sleep statement into smaller batches and recheck time every 10s or so.

Reply:
I have fixed this now. It should always work for the first time the ganache e2e test is executed but if the test is executed mutliple times without `docker-compose down` in between then it can fail.